lms/static/sass/course.scss

```diff
@@ -1,15 +0,0 @@
-/*
```

Comment:
Thanks for the ping @ormsbee . I actually _just_ got back from an extended holiday break, so give me a day or so to page this in :P